symfony/plugins/orangehrmAdminPlugin/config/routes.yaml

```diff
@@ -433,3 +456,18 @@ apiv2_admin_pay_grade:
   methods: [ GET, PUT ]
   defaults:
     _api: OrangeHRM\Admin\Api\PayGradeAPI
+
+view_work_shift:
+  path: /admin/viewWorkShifts
```

Comment:
`/admin/workShift`

Reply:
i added that already, seems cache problems